src/views/SystemIntake/ContractDetails/index.tsx

```diff
@@ -165,77 +165,75 @@ const ContractDetails = ({
                       value
                     />
                     {values.fundingSource.isFunded && (
-                      <div className="margin-top-neg-2 margin-left-3 margin-bottom-1">
-                        <div className="width-card-lg">
-                          <FieldGroup
-                            scrollElement="fundingSource.source"
+                      <div className="margin-top-neg-2 margin-left-4 margin-bottom-1">
+                        <FieldGroup
+                          scrollElement="fundingSource.source"
+                          error={!!flatErrors['fundingSource.source']}
+                        >
+                          <Label htmlFor="IntakeForm-FundingSource">
+                            Funding Source
+                          </Label>
+                          <FieldErrorMsg>
+                            {flatErrors['fundingSource.source']}
+                          </FieldErrorMsg>
+                          <Field
+                            as={DropdownField}
```

Comment:
Shouldn't this width of the DropdownField be restricted to `width-card-lg` like the Funding Number field? You'll need to update the DropdownField component to accept a `className` prop and add the class name here.

Reply:
I did that originally but one of the options was too long and it got cut off by the limited space:

![image](https://user-images.githubusercontent.com/68014929/112213334-5805fa80-8bf4-11eb-98ac-8128ea69db6b.png)

vs.

![image](https://user-images.githubusercontent.com/68014929/112213421-72d86f00-8bf4-11eb-9d9c-90f90d3ca278.png)